server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/oms/openapi/ProjectOpenApiService.java

```diff
@@ -30,6 +32,15 @@
  * @since 4.2.0
  */
 public interface ProjectOpenApiService {
+
+    /**
+     * 查询项目
+     *
+     * @param request
+     * @return
+     */
+    List<ProjectResponse> listProjects(ListProjectRequest request);
```

Comment:
Is this interface currently unavailable anywhere?

Reply:
it's method be called in cloud project